internal/dataexchange/ffdx/config.go

```diff
@@ -24,9 +24,12 @@ import (
 const (
 	// DataExchangeManifestEnabled determines whether to require+validate a manifest from other DX instances in the network. Must be supported by the connector
 	DataExchangeManifestEnabled = "manifestEnabled"
+	// DataExchangeReInitEnabled instructs FireFly to always post all current nodes to the /init API before connecting or reconnecting to the connector
+	DataExchangeReInitEnabled = "reInitEnabled"
```

Comment:
I think this should be `initEnabled` - as you should always set this whenever you have a `ffdx` plugin that supports init. So we don't need to indicate the `re` (which is a bit confusing)

Reply:
Another spelling could be:
`"apiVersion": "1.1"`

Reply:
I think `initEnabled` is appropriate. Personally I think it should be independently toggleable regardless of the plugin version.